### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Painatalman** [Painatalman/yamcha-facts#4]
There are a few accessibility issues that can be improved:

- labels in buttons with only icons
- user should be notified of the new quote without having to go back (aria-live?)
